src/components/side_panel/chart/bar_chart/bar_chart_config_panel.xml

```diff
@@ -14,6 +14,18 @@
             Stacked barchart
           </label>
         </div>
+        <div class="o-checkbox">
+          <label>
+            <input
+              type="checkbox"
+              name="aggregated"
+              t-att-checked="props.definition.aggregated"
+              t-on-change="onUpdateAggregated"
+              class="align-middle"
+            />
+            Aggregate labels
```

Comment:
I'm not sure about the words "Aggregate labels"
Because you are not aggregating labels. You are *aggregating values based on labels**
What about just "Aggregate" ?

We could also add something like `title="Aggregate values with the same label"` on the label element

Reply:
Yep agreed. It makes sense.

Reply:
@Francois-frgi @LucasLefevre I'll add "Aggregate" checkbox just under categories/labels input (I saw in GSheet they place it in a similar way).